### PR TITLE
Add sound on spawnLightning and sendLightning

### PR DESCRIPTION
**Pull request by KennFatt**
### Description

Add sound on spawnLightning and sendLightning
Implements Lightning Sound when Lightning has been called or spawned.

### Reason to modify

For a while, we can use this and wait till we found the real of Lightning Sound from MCPE Client.
And, default Minecraft is has Lightning sound, so why not we try to add this?

### Tests & Reviews

i've been tested it and it seem good

### CODE REVIEW

File: src/pocketmine/level/Level.php [hunk 1]
```diff
@@ -2455,6 +2456,7 @@ public function setChunk(int $chunkX, int $chunkZ, FullChunk $chunk = null, bool
 	 * @param Player $p
 	 */
 	public function sendLighting(int $x, int $y, int $z, Player $p){
+		$this->addSound(new ExplodeSound(new Vector3($x,$y,$z)), $this->getPlayers());
```
Comment (jojoe77777) [inline src/pocketmine/level/Level.php]: > ($x,$y,$z)

Formatting issue - change to:

> ($x, $y, $z)

**Comment by ishitatsuyuki** [comment]
Use a descriptive commit message.

**Comment by KennFatt** [comment]
@ishitatsuyuki yea, i make this commit to make Lightning has a sound, i know its not original source from MCPE's, maybe we can use it for a while till we find the real or original packet sound about Lightning from MCPE Client.

**Comment by ishitatsuyuki** [comment]
Is it a vanilla MCPE behavior? If not, no.

**Comment by KennFatt** [comment]
@ishitatsuyuki i'll find a research for it, i guess is no. just give me a moment...

**Comment by KennFatt** [comment]
http://minecraftpocketedition.wikia.com/wiki/Weather

**Comment by KairusDarkSeeker** [comment]
Worst idea ever in my opinion, just use $level->addSound(Sound);

**Comment by LinuxLogo** [comment]
Ur the worst idea ever... lmao @KairusDarkSeeker

**Comment by KennFatt** [comment]
@KairusDarkSeeker please try your way and give us your code here

**Comment by KairusDarkSeeker** [comment]
@KennFatt Didn't you read the code above?

File: src/pocketmine/level/Level.php (hunk 1 above)
Comment (KairusDarkSeeker) [inline src/pocketmine/level/Level.php]: 👍

**Comment by KennFatt** [comment]
@JackNoordhuis have you tried thunder weather on vanilla (client side) ?

**Pull request closed without merging**